modules/gitops-argo-cd-properties.adoc

```diff
@@ -10,40 +10,101 @@
+  * _<Sharding.replicas>_ - The number of replicas that will be used to support sharding of the ArgoCD Application Controller.
+  * _<Env>_ - Environment to set for the application controller workloads.
+|`DisableAdmin`    |Disables the built-in admin user.|`false` |
+|`GATrackingID`    |Use a Google Analytics tracking ID.|__<empty>__|
+|`GAAnonymizeusers`    |Enable hashed usernames sent to google analytics.|`false` |
+|`HA`    |High availablity options.|__<Object>__
+a|* _<Enabled>_ - Toggle high availability support globally for Argo CD.
+  * _<RedisProxyImage>_ - The Redis HAProxy container image. This overrides the ARGOCD_REDIS_HA_PROXY_IMAGE environment variable.
+  * _<RedisProxyVersion>_ - The tag to use for the Redis HAProxy container image.
+|`HelpChatURL`    |URL for getting chat help (this will typically be your Slack channel for support).|`https://mycorp.slack.com/argo-cd` |
+|`HelpChatText`    |The text appears in a text box for getting chat help.|`Chat now!`|
+|`Image`    |The container image for all Argo CD components. This overrides the `ARGOCD_IMAGE` environment variable.|`argoproj/argocd` |
+|`Ingress`    |Ingress configuration options.|__<Object>__|
+|`InitialRepositories`    |Initial Git repositories to configure Argo CD to use upon creation of the cluster.|__<empty>__|
+|`Notifications`    |Notifications controller configuration options.|__<Object>__
+a|* _<Enabled>_ - The toggle to start the notifications-controller.
+  * _<Image>_ - The container image for all Argo CD components. This overrides the ARGOCD_IMAGE environment variable.
+  * _<Version>_ - The tag to use with the Notifications container image.
+  * _<Resources>_ - The container compute resources.
+  * _<LogLevel>_ - The log level used by the ArgoCD Application Controller component. Valid options are debug, info, error, and warn.
+|`RepositoryCredentials`    |Git repository credential templates to configure Argo CD to use upon creation of the cluster.|__<empty>__|
+|`InitialSSHKnownHosts`    |Initial SSH Known Hosts for Argo CD to use upon creation of the cluster.|__<default_Argo_CD_Known_Hosts>__|
+|`KustomizeBuildOptions`    |The build options and parameters to use with `kustomize build`.|__<empty>__|
+|`OIDCConfig` |The OIDC configuration as an alternative to Dex.|__<empty>__|
+|`NodePlacement` |Add the `nodeSelector` and the `tolerations`.|__<empty>__|
+|`Prometheus` |Prometheus configuration options.|__<Object>__
+a|* _<Enabled>_ - Toggle Prometheus support globally for ArgoCD.
+  * _<Host>_ - The hostname to use for Ingress/Route resources.
+  * _<Ingress>_ - Toggles Ingress for Prometheus.
+  * _<Route>_ - Route configuration options.
+  * _<Size>_ - The replica count for the Prometheus `StatefulSet`.
+|`RBAC` |RBAC configuration options.|__<Object>__
+a|* _<DefaultPolicy>_ - The `policy.default` property in the `argocd-rbac-cm` ConfigMap. The name of the default role which Argo CD will fall back to, when authorizing API requests.
```

Comment:
ConfigMap probably needs to be ticked as in:

 `ConfigMap` resource

or use "config map" on its own

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting